site/content/docs/concepts/environments.en.md

```diff
@@ -29,9 +29,9 @@ When you first create a new environment, no services are deployed to it. To depl
 
 ### VPC and Networking
 
-Each environment gets its own multi-AZ VPC. Your VPC is the network boundary of your environment, allowing the traffic you expect in and out, and blocking the rest. The VPCs Copilot creates are spread across two availability zones to help balance availability and cost - with each AZ getting a public and private subnet.
+Each environment gets its own multi-AZ VPC. Your VPC is the network boundary of your environment, allowing the traffic you expect in and out, and blocking the rest. The VPCs Copilot creates are spread across two availability zones, with each AZ getting a public and private subnet, following [AWS best practices](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-security-best-practices.html). Your services are launched in the public subnets, to help manage costs. Services can be reached only through your load balancer.
```

Comment:
I think we should mention something around security to ensure customers that while it's in public subnets it's still secure 😌.  
We forgot to update this blurb 🙈 , but it's also possible now to create services that are accessible via Service Discovery (`Backend Service`) or an SQS queue (`Worker Service`) so I think we can remove the last piece of the sentence.

```suggestion
Each environment gets its own multi-AZ VPC. Your VPC is the network boundary of your environment, allowing the traffic you expect in and out, and blocking the rest. The VPCs Copilot creates are spread across two availability zones, with each AZ getting a public and private subnet, following [AWS best practices](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-security-best-practices.html). Your services are launched by default in the public subnets while limiting ingress to only other services in your environment for security. Tasks are placed in public subnets to help manage costs by allowing egress to the internet without requiring a NAT gateway.
```